aws/resource_aws_appautoscaling_policy.go

```diff
@@ -286,9 +286,19 @@ func resourceAwsAppautoscalingPolicyCreate(d *schema.ResourceData, meta interfac
 
 func resourceAwsAppautoscalingPolicyRead(d *schema.ResourceData, meta interface{}) error {
 	p, err := getAwsAppautoscalingPolicy(d, meta)
```

Comment:
The API call must be inside the `resource.Retry()` loops to actually retry, otherwise it will continuously just loop with the same error without actually calling the API.